### PR TITLE
Document most VisualServer functions

### PR DESCRIPTION
**Pull request by clayjohn**
Documents ``gi_probe_*``, ``light_*``, ``multimesh_*``, ``particles_*``, ``scenario_*``, and others. 

Tomorrow I will be able to finish up the docs for VisualServer altogether. :)

### CODE REVIEW

**Comment by akien-mga** [comment]
The CI checks picked up two issues with hyperlinks:
```
ERROR: Unresolved member 'gi_probe_set_to_cell_xform', file: VisualServer
ERROR: Unresolved member 'DirectionalLight.directional_shadow_depth_range_mode', file: VisualServer
```

File: doc/classes/VisualServer.xml [hunk 1]
```diff
@@ -1457,6 +1462,7 @@
 			<argument index="0" name="probe" type="RID">
 			</argument>
 			<description>
+				Returns the energy multiplier for this GI Probe. Equivalent to [member GIProbe.energy].
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: I previous descriptions you capitalized as "GI probe", but here (and after) "GI Probe". I'd tend to use "GI probe" everywhere, unless it's an actual reference to the GIProbe node.

File: doc/classes/VisualServer.xml [hunk 2]
```diff
@@ -4459,10 +4552,13 @@
 			The instance is a light.
 		</constant>
 		<constant name="INSTANCE_REFLECTION_PROBE" value="6" enum="InstanceType">
+			The instance is a reflection probe.
 		</constant>
 		<constant name="INSTANCE_GI_PROBE" value="7" enum="InstanceType">
+			The instance is a gi probe.
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: `s/gi/GI/`

File: doc/classes/VisualServer.xml [hunk 3]
```diff
@@ -1481,6 +1489,7 @@
 			<argument index="0" name="probe" type="RID">
 			</argument>
 			<description>
+				Returns the Transform set by [member gi_probe_set_to_cell_xform].
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Should be `method`.

File: doc/classes/VisualServer.xml [hunk 4]
```diff
@@ -2067,6 +2090,7 @@
 			<argument index="1" name="range_mode" type="int" enum="VisualServer.LightDirectionalShadowDepthRangeMode">
 			</argument>
 			<description>
+				Sets the shadow depth range mode for this directional light. Equivalent to [member DirectionalLight.directional_shadow_depth_range_mode]. See [enum LightDirectionalShadowDepthRangeMode] for options.
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Seems like the property is named `DirectionalLight.directional_shadow_depth_range` (without `_mode`).

**Comment by akien-mga** [comment]
The amount of stuff to document for VS is staggering, thanks for working on it! The docs completion rate should increase by a couple percent at least :D

Eventually it might be good to enhance the main description of the VS class to give a clear overview of what those hundreds of methods are meant to be used for. Especially that many are exposed to reproduce the functionality of high level nodes like GIProbe, Particles, etc. without having the overhead of the scene system (IIUC).

**Comment by clayjohn** [comment]
Good thinking! I will include an updated description with my final PR today.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**